.github/workflows/build-testing.yml

```diff
@@ -34,11 +34,11 @@ jobs:
       arch: amd64
       tag: ${{ inputs.tag-prefix }}testing-amd64
       protocol_version_default: 22
-      xdr_ref: v22.0.0
-      core_ref: v22.3.0
-      horizon_ref: horizon-v22.0.2
-      stellar_rpc_ref: v22.1.2
-      friendbot_ref: horizon-v22.0.2
+      xdr_ref: v23.0.0-rc.1
+      core_ref: v23.0.0rc4
+      horizon_ref: horizon-v23.0.0-rc2
+      stellar_rpc_ref: protocol-23
```

Comment:
We can't use branches for these, they need to be tags, because of how the caching works. @stellar/platform-eng Is there a v23 release coming soon for rpc?